rpc/block_svc_routines.c

```diff
@@ -17,7 +17,7 @@
 # include  <netdb.h>
 # include  <uuid/uuid.h>
 # include  <json-c/json.h>
-
+# include  <stdio.h>
```

Comment:
Move this to common.h or utils.h

Reply:
remove as it is already included in utils.h